client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -21,185 +21,208 @@ import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+										'email address. If this is not enabled, a "username" field will display instead.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset>
+								<FormLabel>
+									{ translate( 'Role' ) }
+								</FormLabel>
+								<ReduxFormSelect
 									disabled={ isDisabled }
-									name="applicationMethod"
-									value="url" />
-								<span>
-									{ translate( 'Website URLs only' ) }
-								</span>
-							</FormLabel>
-						</FormFieldset>
-					</Card>
-				</FormSection>
-			</form>
-		</div>
-	);
-};
-
-JobSubmission.propTypes = {
-	isDisabled: PropTypes.bool,
-	translate: PropTypes.func,
-};
+									name="role">
+									<option value="editor">{ translate( 'Editor' ) }</option>
+									<option value="author">{ translate( 'Author' ) }</option>
+									<option value="contributor">{ translate( 'Contributor' ) }</option>
+									<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+									<option value="teacher">{ translate( 'Teacher' ) }</option>
+									<option value="employer">{ translate( 'Employer' ) }</option>
+									<option value="customer">{ translate( 'Customer' ) }</option>
+									<option value="shop_manager">{ translate( 'Shop manager' ) }</option>
+								</ReduxFormSelect>
+								<FormSettingExplanation>
+									{ translate( 'Any new accounts created during submission will have this role. ' +
+										'If you haven\'t enabled account creation during submission in the options above, ' +
+										'your own method of assigning roles will apply.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+						</Card>
+					</FormSection>
+				</form>
+
+				<form>
+					<FormSection name="approval">
+						<SectionHeader label={ translate( 'Approval' ) }>
+							<FormButton compact
+								disabled={ isDisabled }
+								isSubmitting={ isSaving }
+								onClick={ handleSubmit( this.save( 'approval' ) ) } />
+						</SectionHeader>
+						<Card>
+							<FormFieldset>
+								<ReduxFormToggle
+									disabled={ isDisabled }
+									name="isApprovalRequired"
+									text={ translate( 'Require admin approval of all new listing submissions' ) } />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Sets all new submissions to "pending." They will not appear on your ' +
+										'site until an admin approves them.' ) }
+								</FormSettingExplanation>
+
+								<ReduxFormToggle
+									disabled={ isDisabled }
+									name="canEdit"
+									text="Allow editing of pending listings" />
+								<FormSettingExplanation isIndented>
+									{ translate( 'Users can continue to edit pending listings until they are approved by an admin.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+						</Card>
+					</FormSection>
+				</form>
+
+				<form>
+					<FormSection name="duration">
+						<SectionHeader label={ translate( 'Listing Duration' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Duration' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).